parse-opcodes

```diff
@@ -4,6 +4,35 @@ import math
 import sys
 import tokenize
 
+# Instructions not used by the RISC-V Go port (and which should therefore not be
+# emitted).
+GO_UNUSED_INSTRUCTIONS = set((
```

Comment:
Is this necessary for the port? Can it just not use the extra instructions or is there some restriction at play?